pkg/mapper/testworkflows/openapi_kube.go

```diff
@@ -1485,3 +1489,14 @@ func MapTestWorkflowTagSchemaAPIToKube(v testkube.TestWorkflowTagSchema) testwor
 		Tags: v.Tags,
 	}
 }
+
+func MapPvcConfigAPIToKube(v testkube.TestWorkflowPvcConfig) testworkflowsv1.TestWorkflowPvcConfig {
+	return testworkflowsv1.TestWorkflowPvcConfig{
+		Shared:           v.Shared,
```

Comment:
Should there be `shared` option? I think that PVC is in general shared, as long as you know its name

Reply:
yes, the question is that most of pvc storage classes support single pod access mode and they will not work with multiple pods. should we ignore it and leave it tothe user or should we create pvc for each pod, if it's  not `shared``?

Reply:
There is only one execution Pod; by "for each pod" do you mean that such PVC would be shared with multiple executions?

Reply:
Unless you mean the parallel steps and services; in such case `pvcs` should be possible to create for each layer, and accessible all the way down:

```yaml
spec:
  pvcs:
    # created for the whole execution (accessed only by execution)
    oneAtATime:
      storageClassName: standard
    # created for the whole execution (accessed by execution and all services and parallel steps)
    allAtATime:
      storageClassName: standard-many
      accessModes:
      - ReadWriteMany
  pod:
    volumes:
    - name: one-at-a-time
      persistentVolumeClaim:
        claimName: "{{ pvcs.oneAtATime.name }}"
    - name: all-at-a-time
      persistentVolumeClaim:
        claimName: "{{ pvcs.allAtATime.name }}"
  container:
    volumeMounts:
    - name: one-at-a-time
      mountPath: /private
    - name: all-at-a-time
      mountPath: /public
  services:
    count: 3
    pvcs:
      # create for each service instance (accessed only by that instance)
      oneServiceAtATime:
        storageClassName: standard
    pod:
      volumes:
      - name: one-service-at-a-time
        persistentVolumeClaim:
          claimName: "{{ pvcs.oneServiceAtATime.name }}"
      - name: all-at-a-time
        persistentVolumeClaim:
          claimName: "{{ pvcs.allAtATime.name }}"
    container:
      volumeMounts:
      - name: one-service-at-a-time
        mountPath: /private
      - name: all-at-a-time
        mountPath: /public
  steps:
  - parallel:
      count: 3
      pvcs:
        # create for each parallel step (accessed only by that parallel step)
        oneStepAtATime:
          storageClassName: standard
      pod:
        volumes:
        - name: one-step-at-a-time
          persistentVolumeClaim:
            claimName: "{{ pvcs.oneStepAtATime.name }}"
        - name: all-at-a-time
          persistentVolumeClaim:
            claimName: "{{ pvcs.allAtATime.name }}"
      container:
        volumeMounts:
        - name: one-step-at-a-time
          mountPath: /private
        - name: all-at-a-time
          mountPath: /public
```

Reply:
I mean the executtion pod and service pods

Reply:
yes, this makese sense! thank you for suggestion!